en/core-api/e-wallet.md

```diff
@@ -1,76 +1,86 @@
+
+
 # Core API E-Wallet Integration
```

Comment:
@madhura-kulkarni-mh This header structure inconsistency still happen even after we explicitly mentioned on previous feedback?
**Madura's structure**
<img width="257" alt="Screen Shot 2020-11-20 at 11 13 36" src="https://user-images.githubusercontent.com/13027142/99757234-8f1eaf00-2b21-11eb-8f26-d0fddcc957c7.png">

V.S.
**Rashmi's structure**
<img width="255" alt="Screen Shot 2020-11-20 at 11 13 48" src="https://user-images.githubusercontent.com/13027142/99757240-91810900-2b21-11eb-91eb-df7115331f93.png">

Can you agree and coordinate internally please?
cc: @AnujaChangede

Reply:
This another inconsistency:
<img width="769" alt="Screen Shot 2020-11-20 at 11 30 40" src="https://user-images.githubusercontent.com/13027142/99758367-12410480-2b24-11eb-9dda-e482c9480dd8.png">
<img width="764" alt="Screen Shot 2020-11-20 at 11 30 49" src="https://user-images.githubusercontent.com/13027142/99758376-15d48b80-2b24-11eb-93b1-0b5625a6346b.png">

V.S.

<img width="704" alt="Screen Shot 2020-11-20 at 11 31 00" src="https://user-images.githubusercontent.com/13027142/99758397-21c04d80-2b24-11eb-9fe6-ab0b6b54cf72.png">
<img width="704" alt="Screen Shot 2020-11-20 at 11 31 28" src="https://user-images.githubusercontent.com/13027142/99758404-24bb3e00-2b24-11eb-8955-8d55bf858778.png">